src/test/java/uk/gov/hmcts/reform/ccd/documentam/service/impl/DocumentManagementServiceImplTest.java

```diff
@@ -1290,6 +1297,24 @@ void checkUserPermissionTest3() {
                                                                              "exception string"));
     }
 
+    private AuthorisedServices setupAuthorisedServices() {
+        List<String> caseTypeIds = new ArrayList<>();
+        caseTypeIds.add("BEFTA_CASETYPE_1_1");
+        caseTypeIds.add("BEFTA_CASETYPE_2_1");
+
+        AuthorisedService authorisedService = AuthorisedService.builder()
+            .id(XUI_WEBAPP)
```

Comment:
you can actually use mock for "AuthorisedServices"

Reply:
For this, i needed some values to be set for some tests so the service can be authorised first in order to test the caseTypeId and jurisdictionId, some tests were passing but they werent doing what they were supposed to be doing with just a mock